client/src/product-components/AddToCart/AddToCart.jsx

```diff
@@ -47,39 +57,53 @@ class AddToCart extends React.Component {
     });
   }
 
+  handleAddToCart() {
+    axios.post('/cart', {
+      sku_id: this.props.sizeList[this.state.selectedIndex].sku,
+    })
+    .then(function (response) {
+      console.log(response);
+    })
+    .catch(function (error) {
+      console.log(error);
+    });
+
+  }
+
   render() {
     // console.log(this.state.quantityList, '================this.state.quantityList');
-    // console.log(this.props.sizeList, '================this.props.sizeList');
+    // console.log(this.props.sizeList, this.state.selectedIndex, '================this.props.sizeList');
     //console.log(this.props.sizeList, this.state.selectedIndex, 'in render');
     return (
       <div>
-        <div className="SizeSelector">
+        <div  className="SizeSelector">
           Size:
-          <select value={this.state.sizeValue} onChange={this.handleSizeChange.bind(this)}>
+          <select value={this.state.sizeValue} disabled={this.props.sizeOutOfStock} onChange={this.handleSizeChange.bind(this)}>
+            <option value={'SelectSize'}>{this.props.sizeOutOfStock ? 'OUT OF STOCK' : 'Select Size'}</option>
             {
               this.props.sizeList.map((item, index) => {
-              return (
-                <option key={index} value={item.size}>{item.size}</option>
-              )
+                return (
+                  <option key={index} value={item.size}>{item.size}</option>
+                )
             })}
           </select>
         </div><br/>
 
         <div className="QuantitySelector">
           Quantity:
           <select value={this.state.quantityValue} onChange={this.handleQuantityChange.bind(this)}>
-            {
+            {this.state.sizeValue === 'SelectSize' ?  (<option value={'-'}> - </option>) :
               this.QuantityMaxList(this.props.sizeList[this.state.selectedIndex].quantity).map((quantity, index) => {
```

Comment:
By convention only classes are upper-case. The exception are functional components which are also capitalized when imported.